README.md

```diff
@@ -32,8 +33,8 @@ byte-critical systems.
 Axel supports HTTP, HTTPS, FTP and FTPS protocols.
 
 Axel was originally developed by Wilmer van der Gaast. Thanks for your
-efforts. Over time, Axel got several contributions from people. Please,
-see AUTHORS and CREDITS files in source code.
+effort. Over time, Axel has had many contributions from people. Please
```

Comment:
The wording of this line is still a little bit weird.

Reply:
Hmm, maybe just something like:
> Axel was originally developed by Wilmer van der Gaast.
> It has a long history of development and thus had several maintainers and received contributions from many individuals over the years.
> You can find authorship information in the `AUTHORS` and `CREDITS` files, and details in the `ChangeLog` file.

Reply:
Please ship it all in a single commit.

Reply:
I'll edit the commit message if necessary, but try to follow the existing patterns. You can use the "doc:" prefix when editing documentation.